src/types/projectTypes.ts

```diff
@@ -16,7 +15,17 @@ export interface ProjectData {
   description: BlocksContent
   cover_image: ImageResponse
   launch_date: string
-  events: EventResponse[]
+  events: {
+    id: number
+    title: string
+    createdAt: string
+    updatedAt: string
+  }[]
```

Comment:
I think we could do an Omit here:

```js
export interface ProjectData {
  id: number
  title: string
  description: BlocksContent
  cover_image: ImageResponse
  launch_date: string
  events: Omit<EventResponse["data"], "pop_ups">[]
  content_creator: {
    id: number
    artist_name: string
    bio: BlocksContent
  }
}
```

It would tie in the type to our already existing EventResponse like you had done at first and it just helps us ensure consistency across these

Reply:
I always forget about omit...